repositories/src/main/scala/cromwell/pipeline/datastorage/commit/CreateFileRequest.scala

```diff
@@ -0,0 +1,9 @@
+package cromwell.pipeline.datastorage.commit
+
+import play.api.libs.json.{ Json, OFormat }
+
+case class CreateFileRequest(branch: String, content: String, commit_message: String)
```

Comment:
We use camelCase in Scala instead of snake_case

Reply:
Do I need to change formatter, because I need commit_message parameter in Json?

Reply:
Yep